components/resources/brave_components_strings.grd

```diff
@@ -379,7 +379,7 @@
       <message name="IDS_BRAVE_REWARDS_LOCAL_ADS_CONFIRMATION_TYPE_DISMISS" desc="">Dismissed</message>
       <message name="IDS_BRAVE_REWARDS_LOCAL_ADS_CONFIRMATION_TYPE_LANDED" desc="">Clicked</message>
       <message name="IDS_BRAVE_REWARDS_LOCAL_ADS_CONFIRMATION_TYPE_VIEW" desc="">Viewed</message>
-      <message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_NOT_ALLOWED" desc="">BAT is not currently supported in your region. Please refer <ph name="BEGIN_LINK_REGION_TEXT">&lt;a target="_blank" href="$1"&gt;</ph>here<ph name="END_LINK_REGION_TEXT">&lt;/a&gt;</ph> for updates.</message>
+      <message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_NOT_ALLOWED" desc="">BAT is not currently supported in your region. Please refer <ph name="LINK_BEFORE">$1</ph>here<ph name="LINK_AFTER">$2</ph> for updates.</message>
```

Comment:
It looks like we have a couple of different naming conventions - `X_BEFORE/X_AFTER`, and `X_BEGIN/X_END`.  I'd probably suggest `X_BEGIN/X_END` for consistency.

Reply:
cc @petemill. i noticed the inconsistency but went with BEFORE/AFTER because it was what pete suggested in https://bravesoftware.slack.com/archives/C7VLGSR55/p1599801444226400?thread_ts=1599763596.218200&cid=C7VLGSR55. if it's the same either way, i can change it to begin/end.

Reply:
BEGIN/END is fine by me and I agree makes more sense in this context. The text that comes before the BEGIN is the "before", so that's what I've called it in code.

Reply:
thanks! i'll change these to BEGIN_X/END_X to match the existing ones